clrdefinitions.cmake

```diff
@@ -134,7 +134,9 @@ add_definitions(-DFEATURE_MERGE_CULTURE_SUPPORT_AND_ENGINE)
 
 if(CLR_CMAKE_TARGET_ARCH_ARM64)
   # TODO_DJIT: Remove this as part of enabling cross-compiling standalone JIT binary.
-  set(FEATURE_MERGE_JIT_AND_ENGINE 1)
+  if (NOT CLR_CMAKE_PLATFORM_UNIX)
+    set(FEATURE_MERGE_JIT_AND_ENGINE 1)
+  endif(NOT CLR_CMAKE_PLATFORM_UNIX)
 endif(CLR_CMAKE_TARGET_ARCH_ARM64)
```

Comment:
This would break crossgen that is generated for arm64. For arm64 we generate x64 crossgen.exe binary(placed at  bin\Product\Windows_NT.arm64.Debug\x64) which generates arm64 codegen. For that crossgen to be able to work with standalone clrjit.dll we would need to generate x64 clrjit.dll that can generate arm64 codegen. This work has not been done it. This is being tracked by issue #4717